clang/test/Driver/sycl-device-lib-bfloat16.cpp

```diff
@@ -68,6 +68,79 @@
 // RUN:   --sysroot=%S/Inputs/SYCL -### 2>&1 \
 // RUN:   | FileCheck %s -check-prefix=BFLOAT16-FALLBACK-FALLBACK
 
+
+// Test test AOT-DG2 compilation uses native libs + native libs.
```

Comment:
```suggestion
// Test AOT-DG2 compilation uses native libs + native libs.
```

Reply:
> native libs + native libs

Can you add a more descriptive comment describing this part. It is not very clear to me what `native libs + native libs` means.